From 9d748eb141ebbdc39b8969f558c2a83391a39a77 Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Sun, 27 Sep 2026 15:43:07 +0300
Subject: [PATCH] Merge tag 'LA.VENDOR.16.2.0.r1-20500-kaanapali.0' into staging/lineage-24.0_merge-LA.VENDOR.16.2.0.r1-20500-kaanapali.0

LA.VENDOR.16.2.0.r1-20500-kaanapali.0

* tag 'LA.VENDOR.16.2.0.r1-20500-kaanapali.0':
  vibrator: move transient trigger to post-fs-data

Change-Id: Ia7b8cdac65594fc6892de66b7496a68fe0b28d75
---

diff --git a/aidl/vendor.qti.hardware.vibrator.service.rc b/aidl/vendor.qti.hardware.vibrator.service.rc
index 852ed4b..f2b81bd 100644
--- a/aidl/vendor.qti.hardware.vibrator.service.rc
+++ b/aidl/vendor.qti.hardware.vibrator.service.rc
@@ -4,5 +4,5 @@
     group system input wakelock
     capabilities BLOCK_SUSPEND
 
-on late-init
+on post-fs-data
     write /sys/class/leds/vibrator/trigger "transient"
